tests/other_test.go

```diff
@@ -1,2 +1,40 @@
 // Tests in this file are NOT run in the PR pipeline. They are run in the continuous testing pipeline along with the ones in pr_test.go
 package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+)
+
+const basicExampleDir = "examples/basic"
+const ocpVersion1 = "4.17"
```

Comment:
var is named as ocpVersion1 but line 22 is using ocpVersion.
I think that this variable is not being used.